test/unit/date/format.js

```diff
@@ -443,25 +473,100 @@ QUnit.test( "should format various milliseconds (A+)", function( assert ) {
+
+
+	//Test for country with only standard time
+	//eg. long: Indian Standard Time
+	//This test also covers the case where timezone name is undefined 
+	//like short timezone name for Asia/Calcutta and should fall through 'O' format
+	date.isDST = function(){
+		return false;
+	};
+	assert.dateFormatWithTimezone( date, "z", "Asia/Calcutta", cldr, "GMT" );
+	assert.dateFormatWithTimezone( date, "zz", "Asia/Calcutta", cldr, "GMT" );
+	assert.dateFormatWithTimezone( date, "zzz", "Asia/Calcutta", cldr, "GMT" );
+	assert.dateFormatWithTimezone( date, "zzzz", "Asia/Calcutta", cldr, "India Standard Time" );
+
+	date.isDST = function(){
+		return true;
+	};
+	assert.dateFormatWithTimezone( date, "z", "Asia/Calcutta", cldr, "GMT" );
+	assert.dateFormatWithTimezone( date, "zz", "Asia/Calcutta", cldr, "GMT" );
+	assert.dateFormatWithTimezone( date, "zzz", "Asia/Calcutta", cldr, "GMT" );
+	assert.dateFormatWithTimezone( date, "zzzz", "Asia/Calcutta", cldr, "India Standard Time" );
+
+	//fall through to 'O' format
+	//using "America/Argentina/Buenos_Aires" (https://en.wikipedia.org/wiki/America/Argentina/Buenos_Aires)
+	//which is deprecated in CLDR json and thus no metaZones data
+	date = new FakeDate( 0 );
+	assert.dateFormatWithTimezone( date, "z", "America/Argentina/Buenos_Aires", cldr, "GMT" );
+	assert.dateFormatWithTimezone( date, "zz", "America/Argentina/Buenos_Aires", cldr, "GMT" );
+	assert.dateFormatWithTimezone( date, "zzz", "America/Argentina/Buenos_Aires", cldr, "GMT" );
+	assert.dateFormatWithTimezone( date, "zzzz", "America/Argentina/Buenos_Aires", cldr, "GMT" );
+});
+
+QUnit.test( "should format timezone (v)", function( assert ) {
+	var date = new FakeDate( 0 );
+	assert.dateFormatWithTimezone( date, "v", "America/Los_Angeles", cldr, "PT" );
+	assert.dateFormatWithTimezone( date, "vvvv", "America/Los_Angeles", cldr, "Pacific Time" );
+
+	//fall through 'VVVV' format
+	assert.dateFormatWithTimezone( date, "v", "Foo/Bar", cldr, "Foo City Time" );
+	assert.dateFormatWithTimezone( date, "vvvv", "Foo/Bar", cldr, "Foo City Time" );
+
+	//fall through to 'VVVV' format with "Unknown" exemplarCity
+	//using "America/Argentina/Buenos_Aires" (https://en.wikipedia.org/wiki/America/Argentina/Buenos_Aires)
+	//which is deprecated in CLDR json and thus no metaZones data
+	date = new FakeDate( 0 );
+	assert.dateFormatWithTimezone( date, "v", "America/Argentina/Buenos_Aires", cldr, "Unknown City Time" );
+	assert.dateFormatWithTimezone( date, "vvvv", "America/Argentina/Buenos_Aires", cldr, "Unknown City Time" );
+});
+
+QUnit.test( "should format timezone (V)", function( assert ) {
+	var date = new FakeDate( 0 );
+	assert.dateFormatWithTimezone( date, "VV", "America/Los_Angeles", cldr, "America/Los_Angeles" );
+	assert.dateFormatWithTimezone( date, "VVV", "America/Los_Angeles", cldr, "Los Angeles" );
+	assert.dateFormatWithTimezone( date, "VVVV", "America/Los_Angeles", cldr, "Los Angeles Time" );
```

Comment:
Can we include `VVV` and `VVVV` tests that use unkown city? It could re-use the above America/Argentina/Buenos_Aires, but it's good to have it explicitly tested here (in addition to the above fallback-test). Thanks